franco/Project/org.fundacionjala.automationproject/src/main/java/org/fundacionjala/automationproject/ui/PurchasePage.java

```diff
@@ -0,0 +1,224 @@
+  private WebElement deliveryState;
+
+  @FindBy(name = "delZip")
+  private WebElement deliveryPostalCode;
+
+  @FindBy(name = "delCountry")
+  private WebElement deliveryCountry;
+
+  @FindBy(name = "buyFlights")
+  private WebElement buyFlightOption;
+
+  /**
+   * <p>This method clears passengers form fields.</p>
+   */
+  public void clearPassengersFormFields() {
+    passengerName.clear();
+    passengerLastName.clear();
+    creditCardNumber.clear();
+    ccFirstName.clear();
+    ccMiddletName.clear();
+    ccLastName.clear();
+  }
+
+  /**
+   * <p>This method fills the passenger's information.</p>
+   *
+   * @param name      is the passenger name.
+   * @param lastName  is the passenger last name.
+   * @param mealInput is the passenger meal preference.
+   */
+  public void setPassengerInfo(String name, String lastName, String mealInput) {
+    passengerName.sendKeys(name);
+    passengerLastName.sendKeys(lastName);
+    passengerMeal.click();
+    Select meal = new Select(passengerMeal);
+    meal.selectByVisibleText(mealInput);
+  }
+
+  /**
+   * <p>This method fills the credit card information.</p>
+   *
+   * @param cardType   is the credit card type.
+   * @param cardNumber is the credit card number.
+   * @param fName      is the credit card owner name.
+   * @param mName      is the credit card owner middle name.
+   * @param lName      is the credit card owner last name.
+   */
+  public void setCreditCardInfo(String cardType, String cardNumber, String fName, String mName, String lName) {
+    creditCardType.click();
+    Select day = new Select(creditCardType);
+    day.selectByVisibleText(cardType);
+    creditCardNumber.sendKeys(cardNumber);
+    ccFirstName.sendKeys(fName);
+    ccMiddletName.sendKeys(mName);
+    ccLastName.sendKeys(lName);
+  }
+
+  /**
+   * <p>This method clears billing address form fields.</p>
+   */
+  public void clearBillingAddressFormFields() {
+    billingAddress.clear();
+    billingCity.clear();
+    billingState.clear();
+    billingPostalCode.clear();
+  }
+
+  /**
+   * <p>This method fills the billing address information.</p>
+   *
+   * @param bAddress    is the billing address.
+   * @param bCity       is the billing address city.
+   * @param bState      is the billing address state.
+   * @param bPostalCode is the billing address zip.
+   * @param bCountry    is the billing address country.
+   */
+  public void setBillingAddressInfo(String bAddress, String bCity, String bState, String bPostalCode, String bCountry) {
+    billingAddress.sendKeys(bAddress);
+    billingCity.sendKeys(bCity);
+    billingState.sendKeys(bState);
+    billingPostalCode.sendKeys(bPostalCode);
+    billingCountry.click();
+    Select country = new Select(billingCountry);
+    country.selectByVisibleText(bCountry);
+  }
+
+  /**
+   * <p>This method checks if billing address form is not empty.</p>
+   *
+   * @return whether the form is empty or not.
+   */
+  public boolean billingAddressFormIsFilled() {
```

Comment:
You can user this:
return !billingAddress.getText().equals("")    
     && !billingCity.getText().equals("") 
     && !billingState.getText().equals("")
      && !billingPostalCode.getText().equals(""));

Reply:
I removed this method.